inference-engine/tests/functional/plugin/gna/shared_tests_instances/single_layer_tests/activation.cpp

```diff
@@ -22,7 +22,7 @@ const std::vector<InferenceEngine::Precision> netPrecisions = {
         // TODO: Issue:28036
         // InferenceEngine::Precision::FP16,
         InferenceEngine::Precision::I16,
-        InferenceEngine::Precision::I8
```

Comment:
Why int8 was removed here?

Reply:
The plugin reports that it doesn't support I8 input precision.